lib/v2/iwara/subscriptions.js

```diff
@@ -8,40 +8,70 @@ const md = require('markdown-it')({
 });
 
 module.exports = async (ctx) => {
+    if (!config.iwara || !config.iwara.username || !config.iwara.password) {
+        throw Error('Iwara subscription RSS is disabled due to the lack of <a href="https://docs.rsshub.app/install/#pei-zhi-bu-fen-rss-mo-kuai-pei-zhi">relevant config</a>');
+    }
+
     const rootUrl = `https://www.iwara.tv`;
+    const username = config.iwara.username;
+    const password = config.iwara.password;
+
+    // get token
+    const authHeaders = await ctx.cache.tryGet(
+        'iwara:token',
+        async () => {
+            const loginResponse = await got({
+                method: 'post',
+                url: 'https://api.iwara.tv/user/login',
+                headers: {
+                    'content-type': 'application/json',
+                },
+                data: JSON.stringify({
+                    email: username,
+                    password,
+                }),
+            });
+            return {
+                authorization: 'Bearer ' + loginResponse.data.token,
+            };
+        },
```

Comment:
After some testing with the site, the token you returned here is `refresh_token` which will be expired after 1 month. However, this `refresh_token` is used to generate a short-lived `access_token` from https://api.iwara.tv/user/token.

Apart from that, requests for both `videoSubUrl` and `imageSubUrl` are using `access_token` instead of `refresh_token`.